src/main/java/com/springBoot/project/mysqlConnect/ConnectionManager.java

```diff
@@ -0,0 +1,39 @@
+package com.springBoot.project.mysqlConnect;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+
+public class ConnectionManager {
+	private static final Logger logger = Logger.getLogger(ConnectionManager.class);
+	@Value("${jdbc.url}") private static String URL;
+	@Value("${jdbc.username}") private static String USERNAME;
+	@Value("${jdbc.password}") private static String PASSWORD;
+    private static String driverName = "com.mysql.jdbc.Driver";  
+	
+	private static Connection con;
+
+	/**
+	 * This method create and return a connection
+	 * @return Connection
+	 */
+    public static Connection getConnection() {
+        try {
+            Class.forName(driverName);
+            try {
+                con = DriverManager.getConnection(URL, USERNAME, PASSWORD);
```

Comment:
Instead of using nested `try-catch` block you can use only one and `SQLException` into general `catch` block.